plugins/DevicesDetection/Columns/ClientType.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\DevicesDetection\Columns;
+
+use Piwik\Metrics\Formatter;
+use Piwik\Tracker\Request;
+use Piwik\Tracker\Visitor;
+use Piwik\Tracker\Action;
+
+class ClientType extends Base
+{
+    protected $columnName = 'config_client_type';
+    protected $columnType = 'TINYINT( 10 ) NULL DEFAULT NULL';
```

Comment:
@sgiehl I suppose `TINYINT (10)` can't quite work since it stores max 255 different values?

Reply:
sure. Actually meant to use `TINYINT( 1 )`. Will change that

Reply:
👍